ansible_collection/README.md

```diff
@@ -0,0 +1,14 @@
+# HPE Ansible Collections
+
+There are currently 1 HPE Collections
+* [Nimble](https://galaxy.ansible.com/hpe/nimble)
+
```

Comment:
Please keep this in the context of HPE Nimble Storage. Since we don't have the collection submitted, please hold on advertising this URL.

Reply:
ok will rename 
`# HPE Ansible Collections` 
to 
`# HPE Nimble Storage Ansible Collections`

So how about removing the README as of now all together..?? we can push it to repo at the end ??